locale/core.yml

```
@@ -135,20 +135,20 @@ core:
     permissions:
       allow_post_editing_label: Allow post editing
       allow_renaming_label: Allow renaming
-      create_heading: Create
+      create_heading: Vytvořit
```

Nadpisy sekcí dávají více smyslu jako podstatné jméno.

```suggestion
      create_heading: Vytváření
```